sabnzbd/cfg.py

```diff
@@ -92,6 +92,7 @@ def validate_server(value):
 enable_filejoin = OptionBool('misc', 'enable_filejoin', True)
 enable_tsjoin = OptionBool('misc', 'enable_tsjoin', True)
 enable_par_cleanup = OptionBool('misc', 'enable_par_cleanup', True)
+prospective_par_download = OptionBool('misc', 'prospective_par_download', True)
```

Comment:
Maybe we could combine this with the "enable_all_par" option.
But why should it be an option at all? Just always on?